docs/local.md

```diff
@@ -30,7 +56,7 @@ Warning: the command below will kill running instances of `ganache`, `monerod`,
 To avoid confusion, delete any data directories from old runs of `swapd` that used
 the flags `--dev-xmrtaker` or `--dev-xmrmaker`:
 ```bash
-rm -rf /tmp/xmrtaker-* /tmp/xmrmaker-*
+rm -rf "${TMPDIR:-/tmp}"/xmr[mt]aker-*
```

Comment:
The above will use the value of TMPDIR, which is always defined on macOS, or `/tmp` if TMPDIR is not defined.